### PR TITLE
Finish League Challengers Tracker

### PR DESCRIPTION
**Pull request by m-talbi**
# Summary
- Added Main page with statistics representing different regions.
- Added Region page to highlight some of the top challenger players.
- Added Player page to show more info about a specific player and to show the last games that he played.
- Added integration test.
- Added tests for Redux actions.

[Link to video](https://streamable.com/kdeld5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,213 @@
...
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for project inspiration.
+
```
Comment (omarsalem7) [inline README.md]: - [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Kweeka1 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The project is deployed properly ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <div className="header">
+      <div className="mobile-options">
+        <div className="left-side">
+          <RxDotsHorizontal />
+          <span>BELL</span>
+          <AiOutlineWifi />
+        </div>
+        <div className="center">
+          <span className="time">{currentTime}</span>
+        </div>
```
Comment (elisha2kyakpo1) [inline src/Components/Header/Header.jsx:44]: - [ ] [OPTIONAL] This top part should come from the phone of every user using the app.

react

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Kweeka1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

✔️ You have done a great job working on the change(s) requested by the previous reviewer.

## Optional suggestions

- [ ] According to the project requirements, you were supposed to use gitflow (feature branch into development). In future projects, pay attention to this requirement

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**